test/verify/check-superuser

```diff
@@ -56,6 +56,19 @@ class TestSuperuser(testlib.MachineCase):
         b.leave_page()
         b.check_superuser_indicator("Limited access")
 
+        # A reload should not lose privileges
+        b.open_superuser_dialog()
+        b.set_input_text(".pf-v5-c-modal-box:contains('Switch to administrative access') input", "foobar")
+        b.click(".pf-v5-c-modal-box button:contains('Authenticate')")
```

Comment:
Please already call `b.check_superuser_indicator("Administrative access")` here, to wait for the effect of clicking the button. That may fix the test failure, as like this it's a race condition. Moreover, why don't you use `b.become_superuser()`? That will DTRT.